fhir-server-webapp/src/main/liberty/config/bootstrap.properties

```diff
@@ -0,0 +1,3 @@
+javax.xml.stream.XMLInputFactory=com.sun.xml.internal.stream.XMLInputFactoryImpl
+javax.xml.stream.XMLOutputFactory=com.sun.xml.internal.stream.XMLOutputFactoryImpl
+javax.xml.stream.XMLTransformerFactory=com.sun.org.apache.xalan.internal.xsltc.trax.TransformerFactoryImpl
```

Comment:
Can we add a README in the prior location that tells people where to look now?

Reply:
done